resteasy-dependencies-bom/pom.xml

```diff
@@ -881,11 +881,6 @@
                 <artifactId>wildfly-cli</artifactId>
                 <version>${version.org.wildfly.core.wildfly-cli}</version>
             </dependency>
-            <dependency>
-                <groupId>org.wildfly.security</groupId>
-                <artifactId>wildfly-elytron</artifactId>
-                <version>${version.org.wildfly.security.wildfly-elytron}</version>
-            </dependency>
```

Comment:
I guess <version.org.wildfly.security.wildfly-elytron> could be removed ... .

Reply:
@ronsigal Thank you. Fixed